lib/gui/pages/main/main.js

```diff
@@ -25,9 +25,11 @@
 const angular = require('angular');
 const MODULE_NAME = 'Etcher.Pages.Main';
 
+require('angular-moment');
+
 const MainPage = angular.module(MODULE_NAME, [
   require('angular-ui-router'),
-  require('angular-moment'),
+  'angularMoment',
```

Comment:
Maybe this is a great opportunity to start a convention where angular modules that don't return their own module string are put at the top of the dependency array?

For example:

```js
require('angular-moment');

const MainPage = angular.module(MODULE_NAME, [
    'angularMoment',

    require('...'),
    require('...'),
    require('...'),
    require('...')
]);
```